### DISCUSSION TITLE
Regarding the issue of Intant ID face swapping！

### DISCUSSION THREAD

**Discussion by Mdgs668** [lllyasviel/stable-diffusion-webui-forge#2350] [Q&A]
Hello everyone, I am a beginner and I want to implement face swapping in wbeui. I saw a solution that is to use the Intant ID of the control net in the image. I have already downloaded the control net, and according to the tutorial, I found that my preprocessor is correct and has instant_id_face_imbedding, but it is not displayed in my model. Where should I download this model? And put it in that root directory? Please answer, teachers!
![d54195b9104ad3cbd8f683867134a84](https://github.com/user-attachments/assets/98082217-b520-4637-aad3-4f06a7f0a4b6)

**Comment by abzaloff** [comment]
You need to choose this
![Снимок экрана 2024-11-20 163557](https://github.com/user-attachments/assets/92e4de2e-a276-445a-86f7-906b78cbe113)

**Reply by Mdgs668** [reply]
OK！！
